tests/main/uc20-create-partitions-encrypt/task.yaml

```diff
@@ -53,7 +53,7 @@ prepare: |
     echo "Get the UC20 gadget"
     if os.query is-focal; then
         snap download --channel=20/edge pc
-    elif os.query is-ubuntu 22.04 || os.query is-ubuntu 23.04; then
+    elif os.query is-ubuntu 22.04; then
```

Comment:
No need for mantic or noble here? Was this mistakenly tested on lunar before? Not sure why a core test would have run on lunar.

Reply:
This looks like a UC20+ test. Maybe it would make sense for hybrid systems as well, but probably that would require some changes to the test, so for the moment I would not run it for non-UC.

Reply:
ok, thanks for the clarification, I'll create a pr to support uc24 when we start running tests there